sdk/core/System.ClientModel/tests/client/TestClients/SimpleClient/SimpleClientServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace ClientModel.ReferenceClients.SimpleClient;
+
+public static class SimpleClientServiceCollectionExtensions
+{
+    // TODO: How much of these can SCM provide for SCM-based clients to use
+    // from a central source?  Microsoft.Extensions.ClientModel, e.g.?
+
+    // No parameters uses client defaults
+    // See: https://learn.microsoft.com/en-us/dotnet/core/extensions/options-library-authors#parameterless
+    public static IServiceCollection AddSimpleClient(this IServiceCollection services)
```

Comment:
This is the basic end-user extension to add an SCM-based client to a DI service collection.  We'll need to work out some questions like which types of policies do we want to support injection for (all SCM policy implementations?), whether there is an overload that takes a ClientPipelineOptions instance that overrides everything in config and lets users call AddPolicy on it, or whether AddPolicy might be supported via the DI service collection somehow as part of this story?  I'm sure there's more we'll discover on the way.